htdocs/core/class/html.form.class.php

```diff
@@ -4313,7 +4313,7 @@ public function formconfirm($page, $title, $question, $action, $formquestion = '
 							$more .= '<div class="tagtr">';
 							if ($i == 0) $more .= '<div class="tagtd'.(empty($input['tdclass']) ? ' tdtop' : (' tdtop '.$input['tdclass'])).'">'.$input['label'].'</div>';
 							else $more .= '<div clas="tagtd'.(empty($input['tdclass']) ? '' : (' "'.$input['tdclass'])).'">&nbsp;</div>';
-							$more .= '<div class="tagtd'.($i == 0 ? ' tdtop' : '').'"><input type="radio" class="flat'.$morecss.'" id="'.$input['name'].$selkey.'" name="'.$input['name'].'" value="'.$selkey.'"'.$moreattr;
```

Comment:
If you remove $selkey here, several html fields will have the same "id" and this is forbidden by HTML syntax.